100-level-AsyncAPI-Intro/step1.md

```diff
@@ -0,0 +1,54 @@
+Eve assumes her problem-solving role and approaches the spaceship. Inside, Eve discovers Chan, a programming genius visiting from Brownieterra. It becomes apparent that a technical glitch has affected the ship's communication system.
+
+With her expertise in event-driven architecture (EDA) and AsyncAPI, Eve swiftly springs into action. She formulates a plan to establish seamless communication among the spaceship's components in collaboration with Chan.
+
+Eve begins by providing Chan with a comprehensive explanation of event-driven architecture (EDA) and outlines the process of designing an EDA system using AsyncAPI.
+
+## Event-driven architecture
+
+An event signifies a change in the system's state that triggers communication between different services. Event-driven architecture (EDA) decomposes an application into several components interacting via events. These components encompass event producers and consumers:
+
+- An event **producer** identifies an event and transmits relevant information through the event channel.
```

Comment:
Small suggestion. Use `send` instead of transmit since it is part of our ubiquitous language (send/receive). 
```suggestion
- An event **producer** identifies an event and sends relevant information through the event channel.
```